projects/plugins/search/readme.txt

```diff
@@ -1,10 +1,10 @@
 === Jetpack Search ===
-Contributors: automattic, annamcphee, bluefuton, kangzj, jsnmoon
+Contributors: automattic, annamcphee, bluefuton, kangzj, jsnmoon, robfelty, gibrown 
 Tags: search, jetpack
 Requires at least: 6.0
 Requires PHP: 5.6
 Tested up to: 6.1
-Stable tag: 1.2.0
+Stable tag: 1.3.0
 License: GPLv2 or later
 License URI: http://www.gnu.org/licenses/gpl-2.0.html
 
```

Comment:
Let's change the excerpt text also. Right now we aren't using much of the space available: https://wordpress.org/plugins/search/jetpack+search/

Something like:

Easily add instant search and filters to your website or WooCommerce store. Advanced algorithms that boost your search results based on traffic to your site.